Getting Started section rehaul (#28090)
grafana/grafana: docs/sources/getting-started/getting-started.md

+++
title = "Getting started with Grafana"
description = "Guide for getting started with Grafana"
keywords = ["grafana", "intro", "guide", "started"]
type = "docs"
aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
[menu.docs]
identifier = "getting_started-grafana"
parent = "guides"
weight = 200
+++

# Getting started with Grafana

This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).

## Step 1: Install Grafana

Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).

## Step 2: Log in

To log in to Grafana for the first time:

1. Open your web browser and go to http://localhost:3000/. The default HTTP port that Grafana listens to is `3000` unless you have configured a different port.
1. On the login page, enter `admin` for username and password.

> 1. Click **Log In**. If login is successful, you will see a prompt to change the password.

Reviewer: ```suggestion
1. Click **Log In**. If login is successful, then you will see a prompt to change the password.
```

1. Click **OK** on the prompt, then change your password.

> **Note:** We strongly recommend that you follow Grafana's best practices and change the default administrator password. Don't forget to record your credentials!

## Step 3: Create a dashboard

To create your first dashboard:

> 1. Click the **+** icon on the left panel, select **Create Dashboard**, then click **Add new panel**.

Reviewer: ```suggestion
1. Click the **+** icon on the left panel, select **Create Dashboard**, and then click **Add new panel**.
```

Author: Added "then"

Author: Done

1. In the New Dashbard/Edit Panel view, go to the **Query** tab.

> 1. Configure your [query]({{< relref "../panels/queries.md" >}}) by selecting ``-- Grafana --`` from the [data source selector]({{< relref "../panels/queries.md/#data-source-selector" >}}). This generates the Random Walk dashboard.

Reviewer: I don't think you mean dashboard here. Perhaps panel? Or graph?

Author: I went through the steps. It is dashboard.

1. Click the  **Save** icon in the top right corner of your screen to save the dashboard.
1. Add a descriptive name, and then click **Save**.

 Congratulations, you have created your first dashboard and it is displaying results.

## Next steps

 Continue to experiment with what you have built, try the [explore workflow]({{< relref "../explore/index.md" >}}) or another visualization feature. Refer to [Data sources]({{< relref "../features/datasources/data-sources.md" >}}) for a list of supported data sources and instructions on how to [add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). The following topics will be of interest to you:  

- [Panels]({{< relref "../panels/panels-overview.md" >}})
- [Dashboards]({{< relref "../dashboards/_index.md" >}})
- [Keyboard shortcuts]({{< relref "../dashboards/shortcuts.md" >}})
- [Plugins](https://grafana.com/grafana/plugins?orderBy=weight&direction=asc)

### Admins

> The following topics are of interest to team admin or server admin users:

Reviewer: ```suggestion
The following topics are of interest to Grafana server admin users:
```

+++
title = "Getting started with Grafana"
description = "Guide for getting started with Grafana"
keywords = ["grafana", "intro", "guide", "started"]
type = "docs"
aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
[menu.docs]
identifier = "getting_started-grafana"
parent = "guides"
weight = 200
+++

> # Getting Started with Grafana

Reviewer 2: ```suggestion
# Getting started with Grafana
```

Author: Done

> This topic will help you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).

Reviewer 2: ```suggestion
This topic helps you get started with Grafana and build your first dashboard. To learn more about Grafana, refer to [What is Grafana?]({{< relref "what-is-grafana.md" >}}).
```

Author: Done

## Step 1: Install Grafana

Grafana can be installed on many different operating systems. For a list of minimum hardware and software requirements, as well as instructions on installing Grafana, refer to [Install Grafana]({{< relref "../installation/_index.md" >}}).

## Step 2: Log in

> To login to Grafana for the first time:

Reviewer: ```suggestion
To log in to Grafana for the first time:
```

Author: Done

1. Open your web browser and go to http://localhost:3000/. The default HTTP port that Grafana listens to is `3000` unless you have configured a different port.
1. On the login page, enter `admin` for username and password.
1. Click **Log In**.

> 1. If login is successful, you will see a prompt to change the password.

Reviewer: ```suggestion
 If login is successful, then you will see a prompt to change the password.
```

Reviewer: This is not a step. The user is not doing anything.

Author: Fixed, no longer a step.

1. Click **OK** on the prompt, then change your password.

> **Note:** We strongly recommend that you follow Grafana's best practices and change the default administrator password. Don't forget to record your credentials!

## Step 3: Create a dashboard

To create your first dashboard:

1. Click the **+** icon on the left panel, select **Create Dashboard**, then click **Add new panel**.
1. In the New Dashbard/Edit Panel view, go to the **Query** tab.
1. Configure your [query]({{< relref "../panels/queries.md" >}}) by selecting ``-- Grafana --`` from the [data source selector]({{< relref "../panels/queries.md/#data-source-selector" >}}). This generates the Random Walk dashboard.
1. Click the  **Save** icon in the top right corner of your screen to save the dashboard.

> 1. Add a descriptive name, and click **Save**.

Reviewer: ```suggestion
1. Add a descriptive name, and then click **Save**.
```

Author: Done

 Congratulations, you have created your first dashboard and it is displaying results.

## Next steps

>  Continue to experiment with what you have built, try the [explore workflow]({{< relref "../explore/index.md" >}}) or another visualization feature. Refer to [Data sources]({{< relref "../features/datasources/data-sources.md" >}}) for a list of supported data sources and instructions on how to [add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). The following topics will be of interest to you:  

Reviewer: ```suggestion
 Continue to experiment with what you have built, try the [Explore workflow]({{< relref "../explore/index.md" >}}) or another visualization feature. Refer to [Data sources]({{< relref "../features/datasources/data-sources.md" >}}) for a list of supported data sources and instructions on how to [add a data source]({{< relref "../features/datasources/add-a-data-source.md" >}}). The following topics will be of interest to you:  
```

Reviewer: I'm not sure I like the two links removed from the list and inserted into the long paragraph. That might be a little hard to read, and as a user I would either wonder why the links in there are treated differently. Or I might skip past the paragraph down to the list and miss them.

Author: Both are links to specific downloads, that is why I have listed them separately.
